pkg/xray/watches.go

```diff
@@ -0,0 +1,267 @@
+package xray
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/go-resty/resty/v2"
+)
+
+type WatchGeneralData struct {
+	Name        string `json:"name"`
+	Description string `json:"description"`
+	Active      bool   `json:"active"`
+}
+
+type WatchFilterValue struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type WatchFilter struct {
+	Type  string `json:"type"`
+	Value string `json:"value"`
+}
+
+type WatchProjectResource struct {
+	Type            string        `json:"type"`
+	BinaryManagerId string        `json:"bin_mgr_id"`
+	Filters         []WatchFilter `json:"filters"`
+	// Watch a repo
+	Name string `json:"name"`
```

Comment:
I think the comment line confuses the `go fmt`. Either remove the comment (it's a little redundant here), or manually align this line.

Reply:
removed the comment. I'm tired to change it every time manually :)